pkg/sdk/streamlits_gen.go

```diff
@@ -0,0 +1,114 @@
+type CreateStreamlitOptions struct {
+	create       bool                     `ddl:"static" sql:"CREATE"`
+	OrReplace    *bool                    `ddl:"keyword" sql:"OR REPLACE"`
+	streamlit    bool                     `ddl:"static" sql:"STREAMLIT"`
+	IfNotExists  *bool                    `ddl:"keyword" sql:"IF NOT EXISTS"`
+	name         SchemaObjectIdentifier   `ddl:"identifier"`
+	RootLocation string                   `ddl:"parameter,single_quotes" sql:"ROOT_LOCATION"`
+	MainFile     string                   `ddl:"parameter,single_quotes" sql:"MAIN_FILE"`
+	Warehouse    *AccountObjectIdentifier `ddl:"identifier,equals" sql:"QUERY_WAREHOUSE"`
+	Comment      *string                  `ddl:"parameter,single_quotes" sql:"COMMENT"`
+}
+
+// AlterStreamlitOptions is based on https://docs.snowflake.com/en/sql-reference/sql/alter-streamlit.
+type AlterStreamlitOptions struct {
+	alter     bool                    `ddl:"static" sql:"ALTER"`
+	streamlit bool                    `ddl:"static" sql:"STREAMLIT"`
+	IfExists  *bool                   `ddl:"keyword" sql:"IF EXISTS"`
+	name      SchemaObjectIdentifier  `ddl:"identifier"`
+	Set       *StreamlitSet           `ddl:"keyword" sql:"SET"`
+	RenameTo  *SchemaObjectIdentifier `ddl:"identifier" sql:"RENAME TO"`
+}
+
+type StreamlitSet struct {
+	RootLocation *string                  `ddl:"parameter,single_quotes" sql:"ROOT_LOCATION"`
+	MainFile     *string                  `ddl:"parameter,single_quotes" sql:"MAIN_FILE"`
+	Warehouse    *AccountObjectIdentifier `ddl:"identifier,equals" sql:"QUERY_WAREHOUSE"`
+	Comment      *string                  `ddl:"parameter,single_quotes" sql:"COMMENT"`
+}
+
+// DropStreamlitOptions is based on https://docs.snowflake.com/en/sql-reference/sql/drop-streamlit.
+type DropStreamlitOptions struct {
+	drop      bool                   `ddl:"static" sql:"DROP"`
+	streamlit bool                   `ddl:"static" sql:"STREAMLIT"`
+	IfExists  *bool                  `ddl:"keyword" sql:"IF EXISTS"`
+	name      SchemaObjectIdentifier `ddl:"identifier"`
+}
+
+// ShowStreamlitOptions is based on https://docs.snowflake.com/en/sql-reference/sql/show-streamlits.
+type ShowStreamlitOptions struct {
+	show       bool       `ddl:"static" sql:"SHOW"`
+	Terse      *bool      `ddl:"keyword" sql:"TERSE"`
+	streamlits bool       `ddl:"static" sql:"STREAMLITS"`
+	Like       *Like      `ddl:"keyword" sql:"LIKE"`
+	In         *In        `ddl:"keyword" sql:"IN"`
+	Limit      *LimitFrom `ddl:"keyword" sql:"LIMIT"`
+}
+
+type streamlitsRow struct {
+	CreatedOn      string         `db:"created_on"`
+	Name           string         `db:"name"`
+	DatabaseName   string         `db:"database_name"`
+	SchemaName     string         `db:"schema_name"`
+	Title          sql.NullString `db:"title"`
+	Owner          string         `db:"owner"`
+	Comment        sql.NullString `db:"comment"`
+	QueryWarehouse sql.NullString `db:"query_warehouse"`
+	UrlId          string         `db:"url_id"`
```

Comment:
If we have terse, I think all fields besides created_on, name, database_name, schema_name, and url_id should be nullable. Please add an integration test with terse to validate it.

Reply:
added terse int test